.tern-project

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is this file here on purpose?

Reply:
this is the esLint configuration file which is used by eclipse. I put it in purpose but probably each developer should configure? I can not decide to put it here or not :)

Reply:
If it's something specific to Eclipse then it shouldn't be here.